version.xml

```diff
@@ -0,0 +1,8 @@
+<!-- This file may be overwritten by automation. -->
```

Comment:
Why rename from `.props` to `.xml`?  I don't have a preference but I'm curious why you did it.